content/docs/guides/securing-tcp.mdx

```diff
@@ -21,154 +21,150 @@ description: This guide covers how to use Pomerium to protect TCP services such
 
-# Securing TCP based services
+# Securing TCP-based services
 
-The following guide demonstrates how to use Pomerium's [TCP Proxying](/docs/capabilities/tcp) support with various TCP services such as databases and other non-HTTP protocols. It also covers integration points with them when possible.
-
-The source files from this guide can be found on [GitHub](https://github.com/pomerium/pomerium/tree/main/examples/tcp/).
+Learn how to use Pomerium's [TCP Proxying](/docs/capabilities/tcp) support to connect to TCP services, such as databases and non-HTTP protocols.
 
 ## Background
 
-When replacing a traditional VPN, there are often non-HTTP based applications which must still be reachable. Pomerium is able to provide the same type of protection to these services by using a client side application to proxy TCP connections. Authentication and authorization configuration is shared with standard HTTP routes, and the underlying transport is still encrypted between the end-user and Pomerium.
-
-Important notes:
-
-- Pomerium authorizes HTTP on a request-by-request basis, but TCP is authorized on a per-connection basis.
-- Pomerium is only authorizing the TCP _connection_. It does not interact with application level authorization systems at this time.
-
-## How it works
+When replacing a traditional VPN, there are often non-HTTP based applications you still need to reach. Pomerium can provide the same type of protection to these services with [Pomerium CLI](/docs/deploy/clients/pomerium-cli), a client-side application to proxy TCP connections.
 
-- Create a standard Pomerium configuration for your [identity provider (IdP)](/docs/identity-providers)
-- `pomerium-cli` runs on your workstation, listening on loopback for TCP connections
-- When an inbound connection is made, `pomerium-cli` proxies the connection through `pomerium`, authenticating the user if needed
-- Pomerium authorizes the connection and forwards it to the upstream service
-- The connecting application functions as normal
+Authentication and authorization configuration is shared with standard HTTP routes, and the underlying transport is still encrypted between the end-user and Pomerium.
 
-## Pre-requisites
+:::note
 
-This recipe is designed to run on a local docker-compose instance. The included configuration can be adopted for any TCP service, however.
+- Pomerium authorizes TCP connections on a **per-connection basis** (unlike HTTP requests, which it authorizes on a per-request basis)
+- Pomerium only authorizes the TCP _connection_; it does not interact with application level authorization systems
 
-- docker
-- docker-compose
-- A copy of the [example repo](https://github.com/pomerium/pomerium/tree/main/examples/tcp/) checked out
-- Valid credentials for your OIDC provider
-- The [Pomerium Client](/docs/deploy/clients/pomerium-cli) installed
-- (Optional) `mkcert` to generate locally trusted certificates
+:::
 
-## Certificates (optional)
```

Comment:
I think it might be good to keep this section (but maybe skip the part about "This demo comes with its own certificates..."), and have a shorter note below about `--disable-tls-verification`. I think we should encourage users to set up TLS certificates wherever possible, and I don't think installing `mkcert` is much more complicated than installing `pomerium-cli`.